x-pack/plugins/ml/common/types/feature_importance.ts

```diff
@@ -4,8 +4,10 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+export type FeatureImportClassName = string | number | boolean;
```

Comment:
Seeing as you exporting this, would `FeatureImportanceClassName` be a clearer name for this type?

Reply:
Updated here https://github.com/elastic/kibana/pull/80229/commits/19ff8775376d0170175b4e11680490a16e378607